include/epiworld/model-meat.hpp

```diff
@@ -2706,6 +2720,61 @@ inline void Model<TSeq>::draw(
 
 }
 
+template<typename TSeq>
+inline void Model<TSeq>::get_hospitalizations(
+    std::vector< int > & date,
+    std::vector< int > & virus_id,
+    std::vector< int > & counts
+) const
```

Comment:
I think this implementation should be virtual. In some inherited models, we know exactly how to compute this, so we can avoid doing the whole regex work. I like a generic implementation, but give the option of implementing something different.